openhtf/__init__.py

```diff
@@ -45,6 +46,106 @@ class InvalidTestPhaseError(Exception):
   """Raised when an invalid method is decorated."""
 
 
+def Test(*phases, **metadata):
+  return TestInfo(phases, metadata=metadata)
+
+
+class TestInfo(mutablerecords.Record(
```

Comment:
I'm dealing with something at work right now, but I'm good with the second route (`Test` class stores a `TestInfo` record and does the `Execute()`). The point was to enable a different wrapper to create `TestInfo` instances and replace the parts of the actual execution that it wants without needing OpenHTF to support/maintain it.